.circleci/config.yml

```diff
@@ -22,61 +22,77 @@ jobs:
             - v1-npmdeps-
 
       # - run: nix-shell --run 'npm install'
-      - run: yarn install
+      - run: npm install
 
       - save_cache:
           paths:
             - node_modules
           key: v1-npmdeps-{{ checksum "package.json" }}
       
-      - persist_to_workspace:
-          # Must be an absolute path, or relative path from working_directory. This is a directory on the container which is 
-          # taken to be the root directory of the workspace.
-          root: .
-          # Must be relative path from root
-          paths:
-            - node_modules
+      # - persist_to_workspace:
+      #     # Must be an absolute path, or relative path from working_directory. This is a directory on the container which is 
+      #     # taken to be the root directory of the workspace.
+      #     root: .
+      #     # Must be relative path from root
+      #     paths:
+      #       - node_modules
 
-      - run: yarn test
+      - run: npm run test:unit
 
   integration-tests:
     <<: *defaults
 
     steps:
-      - run: sudo apt-get update
-      - run: sudo apt-get install libssl-dev
       - checkout
       - attach_workspace:
           at: ~/project
 
       - restore_cache:
           keys:
-            - holochain-v0.0.9-alpha
+            - holochain-v0.0.18-alpha1
 
-      - run: wget https://github.com/holochain/holochain-rust/releases/download/v0.0.9-alpha/conductor-v0.0.9-alpha-x86_64-ubuntu-linux-gnu.tar.gz
-      - run: wget https://github.com/holochain/holochain-rust/releases/download/v0.0.9-alpha/cli-v0.0.9-alpha-x86_64-ubuntu-linux-gnu.tar.gz
+      - run: wget https://github.com/holochain/holochain-rust/releases/download/v0.0.18-alpha1/conductor-v0.0.18-alpha1-x86_64-generic-linux-gnu.tar.gz
```

Comment:
Can we turn this (version) into variable used here and below? Not sure how yaml works...

Reply:
@danekszy I don't know of a way either. But this circle config may get rewritten soon with @thedavidmeister's help, to use holonix to actually build these binaries instead of downloading them